### PR TITLE
Preserve data

### PR DESCRIPTION
**Pull request by BenMukebo**
## In this Milestone, you will preserve the data from your app and load preserved data on startup.

- [ ] Check the documentation for Ruby's JSON library.
- [ ] On exit from your app save all the data (books, people, and rentals) into JSON files. One per type of data you are storing and named accordingly (e.g. rentals.json)
- [ ] On startup of your app load all data from files. Remember to handle the case when one or more of the files are missing.

### CODE REVIEW

File: .rubocop.yml [hunk 1]
```diff
@@ -22,6 +22,8 @@ Metrics/CyclomaticComplexity:
   Enabled: false
 Metrics/MethodLength:
   Enabled: false
+Metrics/ClassLength:
+  Enabled: false
```
Comment (Dipeshtwis) [inline .rubocop.yml]: - [x] I would not recommend editing the `rubocop.yml` file. Please use the file without editing or updating. You can find this [here](https://github.com/microverseinc/linters-config/blob/master/ruby/.rubocop.yml).

**Review by Dipeshtwis — CHANGES_REQUESTED** [review]
Hi @BenMukebo & @mmsesay, 

:+1: This is good to make it this far, your project is really nice, congratulations on the good work you have done so far.
:+1: I can store and retrieve json file.

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


 ## Required Changes ♻️

**_Check the comments under the review_**

- [ ] I would recommend including the steps to run the project. [Hint: `ruby main.rb`].

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

- N.A.

Cheers and Happy coding!👏👏👏
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**_Dipesh Kumar_**


------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project(check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: .rubocop.yml (hunk 1 above)
Comment (mmsesay) [reply .rubocop.yml]: Thank you very much @Dipeshtwis , we refactor our code and remove the added linter config line.

**Review by Emmyn5600 — APPROVED** [review]
STATUS: APPROVED :+1: 

Hello, @BenMukebo and @mmsesay   👋🏾 👋🏾 congratulations on implementing all the project requirements. All classes and methods have been implemented successfully.

## Optional suggestions

- N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

**Pull request merged**